src/scrabble.js

```diff
@@ -1,5 +1,49 @@
-function scrabble() {
-
+function scrabble(string) {
+  if (string === null) {
+    return 0
+  } else {
+    const word = string.toUpperCase()
+    let value = 0
+    for (let i = 0; i <= word.length; i++) {
+      if (
```

Comment:
You could create an object that holds all of the characters and their specific values, that way you only need to check if the character at index i is included in the object and if true you could then add that characters value to the value with something like `value += objectname[word[i]]`